fdbclient/SystemData.cpp

```diff
@@ -387,6 +413,37 @@ StorageServerInterface decodeServerListValue( ValueRef const& value ) {
 	return s;
 }
 
+
+const KeyRangeRef cacheListKeys(
```

Comment:
```suggestion
// "\xff/cacheList/[[UID]]" := StorageServerInterface
const KeyRangeRef cacheListKeys(
```

Reply:
I am just going to remove cacheListKeys as we now have storageCacheServerKeys.